spotifycli/spotifycli.py

```diff
@@ -82,32 +85,37 @@ def start_shell():
 def add_arguments():
     parser = argparse.ArgumentParser(description=__doc__)
     for argument in get_arguments():
-        parser.add_argument(argument[0], help=argument[1], action="store_true")
+        if not argument[2]:
+            parser.add_argument(argument[0], help=argument[1], action="store_true")
+        else:
+            parser.add_argument(argument[0], help=argument[1], action="store")
```

Comment:
The `get_arguments` function row returns a list of `tuple[str, str, bool]` where the third argument specifies whether this is a `store` argument or a `store_true` argument.